lib/utils.js

```diff
@@ -118,6 +118,20 @@ const utils = {
   },
 
   /**
+   * Grab the most recent PR
+   * @param {Config} config - the config object
+   * @param {Github} vcs - VCS interface
+   * @returns {Promise} a promise resolved with the most recent PR
+   */
+  getLastPr (config, vcs) {
+    return exec(`git log -50 --oneline | grep -m 1 "Merge pull request" | awk '{print $5;}'`).then((stdout) => {
```

Comment:
But, I figured, what the heck, just in case 49 PRs are merged the second after this one is, before the build starts ;)